CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.1.2 (2019-12-12)
+
+- Fixed the modal component to correctly stack children elements inside the modal. [#73](https://github.com/blackbaud/skyux-modals/pull/73)
```

Comment:
Child instead of children? Only comment I have. Other than that 👍

Reply:
I think children is correct here because it is two child elements in relation to each other. Thoughts? @Blackbaud-DeniseCisneros

Reply:
I read it as "child" modifying "element", so if it needed to be plural, the modifier would be the same but element would be pluralized.

Reply:
OOOOOOHHHHHH. That makes sense. I'll make the change

Reply:
Done